main.tf

```diff
@@ -78,6 +87,28 @@ resource "azurerm_container_group" "linux-container-group" {
       server   = var.image_registry_credential.server
     }
   }
+
+  # identity block
```

Comment:
can you add a comment in the code to describe the 3 use case (only smi, only umi or both). I am afraid we will forgot this in a few weeks :D

Reply:
There is this comment in line 23

```
# umi == user managed identity, smi == system managed identity
```

do you want me to move it, duplicate it, leave it as is or explain something else?

Reply:
I meant explain the 3 use cases, not the acronyms

Reply:
commit 4419c13d3b8b8932427fb6d15c5e416c3162dfc9